packages/components/src/Tooltip/utils/isSemanticElement.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import { Button, IconButton } from "@kaizen/button"
+import { Icon } from "@kaizen/component-library"
+import ArrowIcon from "@kaizen/component-library/icons/arrow-right.icon.svg"
```

Comment:
We don't want to bring `@kaizen/component-library` into KAIO, so best to refactor these usages.

Reply:
`<ArrowRightIcon />`